src/main/java/com/github/alinaberlin/security/controlers/AuthController.java

```diff
@@ -0,0 +1,56 @@
+package com.github.alinaberlin.security.controlers;
+
+
+import com.github.alinaberlin.security.components.JWTEncoderDecoder;
+import com.github.alinaberlin.security.models.User;
+import com.github.alinaberlin.security.payload.request.LoginRequest;
+import com.github.alinaberlin.security.payload.response.ErrorRes;
+import com.github.alinaberlin.security.payload.response.LoginRes;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```

Comment:
We should make this `@RestController`. Because you annotated it as `@Controller` you had to use the `@ResponseBody` annotation for your function